### ISSUE TITLE
Add package to conda-forge

### ISSUE THREAD

**Issue by dstansby** [hapi-server/client-python#68]
I'm currently playing around with trying to create a `conda` environment that contains all the core PyHC packages. It would be great if hapiclient could be added to `conda-forge`, and I think all it's dependencies are already there so it shouldn't be too hard.

I would be happy to open a PR to add `hapiclient` to `conda-forge` if that would be welcome?

**Comment by dstansby** [comment]
Since it was easy enough to do, I opened a PR at https://github.com/conda-forge/staged-recipes/pull/22148 to add `hapiclient` to conda-forge. I conservatively guessed at a minimum Python version of 3.8, and was wondering if `hapiclient` officially supports older Python versions, and if so how far back support goes?

**Comment by rweigel** [comment]
Sounds good. hapiclient is tested on 2.7, 3.5, ..., 3.10.

I had been avoiding conda-forge because it was another way install issues can arise. But it sounds like you'll be available for questions. Do they pull from pypi or does one need to push a release?

**Comment by dstansby** [comment]
Yes, it gets automatically pulled from pypi. I'm happy to maintain the packaging on conda-forge for now 👍 , but if you (or anyone else on the hapiclient dev team) like to be added as a maintainer let me know.

**Comment by rweigel** [comment]
Is it possible to add your contact information to https://anaconda.org/conda-forge/hapiclient? I think one can click the `cf-staging / hapiclient` link on that page and see the associated repo, from which one can see you as the maintainer (or if you find https://github.com/conda-forge/hapiclient-feedstock). However, viewing the content of that link requires an account on Anaconda.

I had forgotten about this thread, and Bernie Harris asked me about the conda-forge package, and I said there was none. I went to https://anaconda.org/conda-forge/hapiclient and could not find a maintainer's name. Later, Bernie pointed me to https://github.com/conda-forge/hapiclient-feedstock.

**Comment by dstansby** [comment]
I've never come across a way to do that I'm afraid.
